anomalib/models/cfa/lightning_model.py

```diff
@@ -73,43 +74,40 @@ def on_train_start(self) -> None:
         """Initialize the centroid for the memory bank computation."""
         self.model.initialize_centroid(data_loader=self.trainer.datamodule.train_dataloader())  # type: ignore
 
-    def training_step(self, batch) -> STEP_OUTPUT:
+    def training_step(self, batch: dict[str, str | Tensor], *args, **kwargs) -> STEP_OUTPUT:
```

Comment:
Is it really needed to add `*args` and `**kwargs` here and in the other lightning models?

Reply:
> Is it really needed to add `*args` and `**kwargs` here and in the other lightning models?

The original `LightningModule` has this signature. Looks like this is the proper way of inheritance to satisfy pylint. Otherwise pylint throws `arguements-differ` error.